UserReportManager.kt

```diff
@@ -1,3 +1,13 @@
 object UserReportManager {
 
+    val userReportService: UserReportService = UserReportService()
+
+    fun sendAllUsersReports(users: List<User>) {
+        for (user in users) {
+            for (userReport in user.reports.orEmpty()) {
+                userReportService.sendReport(userReport)
+            }
```

Comment:
Did we consider any alternatives to the nested for loops? I’ve really been liking the standard libs collection functions.

Reply:
Thanks! I'm still getting used to the standard lib.